### PR TITLE
Quality of life improvements

### PR DESCRIPTION
**Pull request by Pop0097**

### CODE REVIEW

File: cli/src/command/mod.rs [hunk 1]
```diff
@@ -4,3 +4,43 @@ pub mod list_commands;
 pub mod search_command;
 pub mod search_utils;
 pub mod update_command;
+
+use data::models::InternalCommand;
+use prettytable::{format, Attr, Cell, Row, Table};
+
+pub fn print_internal_command(internal_command: &InternalCommand) {
```
Comment (dylan-p-wong) [inline cli/src/command/mod.rs:11]: Should this method be in another file other than mod.rs?

File: cli/src/main.rs [hunk 2]
```diff
@@ -30,10 +32,10 @@ enum LoggerInitializationError {
     InitializingLogger,
 }
 
-/// Set up logging. Can log in any crate using the log crate
+/// Set up logging for CLI
 ///
 /// If we are doing local development, set up environment logger.
-/// Otherwise (if the app was built with the --release flag), send logs to a file in the user's file statem.
+/// Otherwise (if the app was built with the `--release`` flag), send logs to a file in the user's file statem.
```
Comment (dylan-p-wong) [inline cli/src/main.rs]: Typo

File: cli/src/utils.rs [hunk 3]
```diff
@@ -0,0 +1,9 @@
+pub fn truncate_string(s: &str, width: usize) -> String {
+    if width < 3 {
```
Comment (dylan-p-wong) [inline cli/src/utils.rs]: do we need this branch? What is s == "abc"

**Review by dylan-p-wong — APPROVED** [review]
I see sometimes we use \n sometimes println! sometimes both. Not sure what best practice is in Rust.

File: cli/src/command/mod.rs (hunk 1 above)
Comment (dylan-p-wong) [reply cli/src/command/mod.rs:11]: Left for now. We can refactor later.

File: cli/src/utils.rs (hunk 3 above)
Comment (dylan-p-wong) [reply cli/src/utils.rs]: Fixed and added tests.

**Review by danyal002 — APPROVED** [review]
I'm just gonna LGTM it for now and fix it later.

**Pull request merged**